pynicotine/pluginsystem.py

```diff
@@ -739,42 +735,49 @@ def plugin_settings(self, plugin_name, plugin):
         except KeyError:
             log.add_debug("No stored settings found for %s", plugin.human_name)
 
-    def get_command_descriptions(self, command_interface, search_query=None):
+    def get_autocomplete_commands(self, command_interface):
 
-        command_groups = {}
+        completion_list = []
```

Comment:
`get_autocomplete_commands()` -> `get_command_list()`
`completion_list` -> `command_list`

This function could be used for any purpose.